addons/io_scene_gltf2/__init__.py

```diff
@@ -74,11 +74,11 @@ class ExportGLTF2_Base:
 
     export_format = EnumProperty(
         name='Format',
-        items=(('GLB', 'Binary (.glb)', 'Exports a single file, with all data packed in binary form. Most efficient and portable, but more difficult to edit later'),
-               ('GLTF', 'JSON (.gltf)', 'Exports a single file, with all data packed in JSON. Less efficient than binary, but easier to edit later'),
-               ('GLTF_SEPARATE', 'JSON (.gltf + .bin + textures)', 'Exports multiple files, with separate JSON (.gltf), binary (.bin), and texture data. Easiest to edit later')),
+        items=(('GLB', 'glb', 'Exports a single file, with all data packed in binary form. Most efficient and portable, but more difficult to edit later'),
+               ('GLTF_EMBEDDED', 'glTF embedded)', 'Exports a single file, with all data packed in JSON. Less efficient than binary, but easier to edit later'),
```

Comment:
Got an extra closing parenthesis on `glTF embedded)`.

I'm not sure whether I prefer these names... We are assuming the user knows the difference already? I can see where the term "JSON" is not something an artist wants to know about, but for example the FBX exporter has "Binary" and "ASCII" options similar to this. And a glb is still a glTF file...

/cc @emackey any preferences on this?

Reply:
I'm happy with that, thanks.